lib/active_admin/inputs/filters/date_range_input.rb

```diff
@@ -33,7 +33,7 @@ def input_html_options(input_name = gt_input_name, placeholder = gt_input_placeh
             class: "datepicker",
             maxlength: 10,
             placeholder: placeholder,
-            value: current_value ? current_value.strftime("%Y-%m-%d") : "" }
+            value: current_value ? current_value.strftime("%Y-%m-%d") : "" }.merge(options[:input_html] || {})
```

Comment:
I realized that we are overriding the formtastic `input_html_options` method, but changing its signature (normally, it receives no arguments). So any code in the formtastic base class calling this method will break.

Let's not override it, and call it something else:

```diff
diff --git a/lib/active_admin/inputs/filters/date_range_input.rb b/lib/active_admin/inputs/filters/date_range_input.rb
index 399121ea0..5d32cf075 100644
--- a/lib/active_admin/inputs/filters/date_range_input.rb
+++ b/lib/active_admin/inputs/filters/date_range_input.rb
@@ -7,8 +7,8 @@ module ActiveAdmin
         def to_html
           input_wrapping do
             [ label_html,
-              builder.text_field(gt_input_name, input_html_options(gt_input_name, gt_input_placeholder)),
-              builder.text_field(lt_input_name, input_html_options(lt_input_name, lt_input_placeholder)),
+              builder.text_field(gt_input_name, input_html_options_for(gt_input_name, gt_input_placeholder)),
+              builder.text_field(lt_input_name, input_html_options_for(lt_input_name, lt_input_placeholder)),
             ].join("\n").html_safe
           end
         end
@@ -22,7 +22,7 @@ module ActiveAdmin
           column && column.type == :date ? "#{method}_lteq" : "#{method}_lteq_datetime"
         end
 
-        def input_html_options(input_name = gt_input_name, placeholder = gt_input_placeholder)
+        def input_html_options_for(input_name = gt_input_name, placeholder = gt_input_placeholder)
           current_value = begin
             #cast value to date object before rendering input
             @object.public_send(input_name).to_s.to_date
@@ -33,7 +33,7 @@ module ActiveAdmin
             class: "datepicker",
             maxlength: 10,
             placeholder: placeholder,
-            value: current_value ? current_value.strftime("%Y-%m-%d") : "" }.merge(options[:input_html] || {})
+            value: current_value ? current_value.strftime("%Y-%m-%d") : "" }.merge(input_html_options)
         end
 
         def gt_input_placeholder 
```